GitHub_Learning/Generic/Navigation/View/WireframeImpl.swift

```diff
@@ -42,12 +45,11 @@ class WireframeImpl: Wireframe {
     func launchDashboard(from point: DashboardLaunchPoint) {
         switch point {
         case .login:
-            presentViewControllerModally(DashboardViewController.self, with: .fullScreen, animated: true) {
-                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
-                    self.setViewControllerAsRoot(DashboardViewController.self)
-                }
-            }
-        case .startUp: setViewControllerAsRoot(DashboardViewController.self)
+            let tabbar = makeTabBarController()
+            self.window.rootViewController = tabbar  
+        case .startUp:
+            let tabbar = makeTabBarController()
+            self.window.rootViewController = tabbar
```

Comment:
Tady o tomhle vim. Ale když to presentnu modálně a pak nastavim jako root tak to křičí nějakej error a ještě jsem nepřišel na to jak udělat. Kdyžtak to nechám jen jako root a bez animace ale zkusim to ještě nějak vyřešit.

Reply:
Obe ty vetve delaji to same... proc tam mas switch?

Reply:
Tady jsem ti to komentoval, proč je tam mám..

Reply:
Ale ja se ptam prot tam rozlisujes z jakoho mista se ten dashboard zobrauje. Kdyz tam nebudes mit ten switch tak je to uplne stejne.

Reply:
Já vim, právě píšu, že původně jsem to rozlišoval ale potom co jsem tam dal tabBar tak mám problém že mi to něco řve tak jsem tam zatim dal obě cesty stejný dokud nezjistim jak obejít ten error. Šlo o to že původně jsem to nejdřív zobrazit modálně a pak nastavil jako root (aby tam byla animace u cesty z loginu), pak to začalo křičet a já se snažim najít způsob jak to udělat zase jako předtím (u toho casu .login), proto jsem to zatim nechal stejný pro oba casy, abych to nemusel mazat a pak zase vytvářet.

Reply:
Uz to chapu, tak to je asi potreba zmenit pristup k login obrazovce.